docs/development/affiliated-packages.rst

```diff
@@ -269,7 +269,7 @@ files manually`_ section since this explains what many of the files do.
       or just delete it.
    #. Exit out of your text editor.
 
-#. Update the main package docstring in ``<packagename>/__init__.py``.
+#. Update the main package docstring in ``<packagename>/packagename/__init__.py``.
```

Comment:
@pllim Yeah, but when you follow the instructions, you download the whole repo as <packagename>, i.e. <packagename> on this doc page corresponds to package-template. And then you go one more folder down into packagename.  I can show you on my local copy if you want...